giraffe/src/utils/annotationData.ts

```diff
@@ -35,6 +36,13 @@ const isWithinHoverableArea = (
   )
 }
 
+const distanceToMousePointer = (
+  annotationMark: AnnotationMark,
+  hoverPosition: number
+) => {
+  return Math.abs(hoverPosition - annotationMark.startValue)
```

Comment:
This is great and works when Annotations are a single line.

This won't work when Annotations are ranges though. I think we can make a quick tweak to allow it to work with ranges.

- hoverPosition is considered a distance of 0 from the annotation range if it is between the annotation.start and annotation.stop.
- otherwise hoverPosition has a distance that is shortest of: the distance between hoverPosition and annotation.start, and the distance between hoverPosition and annotation.stop

Reply:
i typed that very quickly. i meant `startValue` and `stopValue` but hopefully you get the idea :smile:

Reply:
Great point Timmy, I always wondered why there was a `startValue` and `stopValue` when they are the same,  but this makes sense. Didn't know Annotation can be a range. I have some homework due in 2 hrs and I'll address this right after that!